src/components/thumbnail.ts

```diff
@@ -55,21 +54,36 @@ export class FrigateCardThumbnailFeatureEvent extends LitElement {
     if (this.thumbnail?.startsWith('data:')) {
       return this.thumbnail;
     }
-    return await homeAssistantSignPath(this.hass, thumbnail);
+    return new Promise((resolve, reject) => {
+      //eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+      this.hass!.fetchWithAuth(thumbnail)
```

Comment:
Stylistic: Can you avoid the non-null-assertion, e.g. just reject the promise if hass is null?

Reply:
For some reason I need to add a `return;` after the `reject();` otherwise the linter complains. I'm a JS novice - can you explain why a `reject()` is not sufficient?

Reply:
What doest the linter say? Probably it's that the following line uses `this.hass` which might be undefined without the return and you're not using `this.hass?` to indicate that is okay if that is the case. By returning, Typescript works out that the following statement will not execute and so does not complain.